head.go

```diff
@@ -1256,9 +1264,10 @@ type memSeries struct {
 	chunkRange   int64
 	firstChunkID int
 
-	nextAt    int64 // timestamp at which to cut the next chunk.
-	lastValue float64
-	sampleBuf [4]sample
+	nextAt     int64 // timestamp at which to cut the next chunk.
+	lastValue  float64
+	sampleBuf  [4]sample
+	addPending bool
```

Comment:
a little comment of its purpose would be good  and what about if we call it `pendingCommit`